common/config/rush/common-versions.json

```diff
@@ -65,6 +65,8 @@
     // projects to choose a higher version if desired.
     "@azure/core-http": ["^1.0.0"],
     // Idenity is moving from v1 to v2. Moving all packages to v2 is going to take a bit of time, in the mean time we could use v2 on the perf-identity tests.
-    "@azure/identity": ["^2.0.0-beta.1"]
+    "@azure/identity": ["^2.0.0-beta.1"],
+    "@microsoft/api-extractor": ["7.13.2"],
+    "prettier": ["2.2.1"]
```

Comment:
The current versions we use don't support Literal String Templates 😢  for prettier I think we'd be able to update across the board. For api-extractor I think there were some issues in the output last time we tried to update.

Since updating prettier in this PR would cause a lot of noise I'm thinking to send a follow up PR

Reply:
sure, can you just file issues and link them from comments?

Reply:
The issue that tracks the effort to upgrade prettier: https://github.com/Azure/azure-sdk-for-js/issues/9329 and the one for api-extractor is https://github.com/Azure/azure-sdk-for-js/issues/9410.